src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1068,6 +1070,19 @@ public static void onAdvancement(ServerPlayerEntity player, Advancement advancem
         MinecraftForge.EVENT_BUS.post(event);
         return event.getModifiers();
     }
+    
+    /**
+     * Hook to fire {@link PlayerFindProjectileEvent}. Returns the ammo to be used.
+     */
+    public static ItemStack findProjectile(PlayerEntity player, ItemStack shootable, ItemStack ammo)
+    {
+        PlayerFindProjectileEvent event = new PlayerFindProjectileEvent(player, shootable, ammo);
+        if (MinecraftForge.EVENT_BUS.post(event)) {
+            return player.isCreative() ? new ItemStack(Items.ARROW) : ItemStack.EMPTY;
+        } else {
+            return event.getProjectile();
```

Comment:
The code style needs to stay consistent and match the style with the rest of the API (check your braces here)